src/object/ambient_sound.hpp

```diff
@@ -59,7 +59,7 @@ class AmbientSound final : public MovingObject,
   virtual HitResponse collision(GameObject& other, const CollisionHit& hit_) override;
 
   virtual std::string get_class() const override { return "ambient-sound"; }
-  virtual std::string get_display_name() const override { return _("Ambient sound"); }
+  virtual std::string get_display_name() const override { return _("Sound"); }
```

Comment:
https://backtracks.fm/resources/podcast-dictionary/ambient+sound
> Ambient sound in relation to audio refers to the background noise present at a given scene or a location. This can include noises such as rain, traffic, crickets, birds, et cetera.

Ambient Sound is more specific than just Sound.